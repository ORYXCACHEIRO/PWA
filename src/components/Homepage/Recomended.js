import teste from '../../assets/exterior.jpg';

const Recomended = () => {

    return (
        <>
            <div className="ml-4 mr-4">
                <div>
                    <h1 className="text-black m-5 ml-8 font-extrabold text-5xl text-center md:text-4xl">Recomendations.</h1>
                </div>
                <div className="inline-flex pt-12 mr-auto ml-auto flex-wrap w-full justify-center gap-16 pb-10"> 
                    <div className="sm:ml-0">
                        <div className="bg-purple-500 h-96 w-96 relative sm:right-2 rounded-lg">
                            <div className="bg-black h-96 w-96 absolute left-7 top-7 rounded-lg">
                                <div className="p-7">
                                    <img src={teste} alt="" className=" rounded-xl"/>
                                </div>
                            </div>
                        </div>
                    </div>
                    <div className="sm:ml-0">
                        <div className="bg-purple-500 h-96 w-96 relative sm:right-2">
                            <div className="bg-black h-96 w-96 absolute  left-7 top-7">

                            </div>
                        </div>
                    </div>
                    <div className="sm:ml-0">
                        <div className="bg-purple-500 h-96 w-96 relative sm:right-2">
                            <div className="bg-black h-96 w-96 absolute  left-7 top-7">

                            </div>
                        </div>
                    </div>
                    <div className="sm:ml-0">
                        <div className="bg-purple-500 h-96 w-96 relative sm:right-2">
                            <div className="bg-black h-96 w-96 absolute  left-7 top-7">

                            </div>
                        </div>
                    </div>
                </div>
            </div>
            
        </>
    )

}

export default Recomended;