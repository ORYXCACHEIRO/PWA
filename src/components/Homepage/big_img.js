import bigimg from '../../assets/roll.jpg';
import Search from '../Header/Search';

const BigImg = () => {

    return(
        <>
            <div className='ml-4 mr-4  relative aspect-[10/5] md:aspect-[10/6] sm:aspect-[9/13] flex items-start flex-col justify-center '>
                <div className='  relative ml-10 lg:ml-0'>
                    <h1 className=' font-bold text-gray-900  text-9xl  md:text-8xl'>Looking for a place?</h1>
                    <h1 className='font-bold text-gray-900  text-9xl  md:text-8xl'>Start <span className='text-purple-500'>now.</span></h1>
                </div>
                
                
                <div className='  h-max w-2/4 sm:w-full xl:w-3/4 mt-10 '>
                    <Search/>
                </div>
            </div>
        </>
    )

}

export default BigImg;