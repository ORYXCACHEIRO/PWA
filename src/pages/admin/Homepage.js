

const Homepage = () => {

    return(
        <>
           <div className="flex flex-wrap border-2 border-green-500 h-full gap-7">
               <div className="border-2 border-blue-500 m-7 px-10 py-5 flex items-center gap-4 rounded-2xl h-max">
                    <h1 className="font-medium text-4xl text-slate-600">Users</h1>
                    <h1 className="font-bold text-4xl text-slate-800">0</h1>
               </div>
               <div className="border-2 border-blue-500 m-7 px-10 py-5 flex items-center gap-4 rounded-2xl h-max">
                    <h1 className="font-medium text-4xl text-slate-600">Users</h1>
                    <h1 className="font-bold text-4xl text-slate-800">0</h1>
               </div>
           </div>
        </>
    )

}

export default Homepage;