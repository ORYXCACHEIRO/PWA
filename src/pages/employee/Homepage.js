const Homepage = () => {

    return(
        <>
        a
        </>
    )

}

export default Homepage;