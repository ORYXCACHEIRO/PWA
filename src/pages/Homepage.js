import Recomended from "../components/Homepage/Recomended";
import BigImg from "../components/Homepage/big_img";

const Homepage = () => {

    return(
        <>
            <BigImg/>
            <Recomended/>
        </>
    )

}

export default Homepage;