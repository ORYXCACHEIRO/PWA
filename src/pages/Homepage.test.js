import React from 'react'
import Enzyme, { shallow, mount } from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17'
import Homepage from './Homepage';
import { Simulate } from 'react-dom/cjs/react-dom-test-utils.production.min';
import renderer from 'react-test-renderer';


Enzyme.configure({ adapter: new Adapter() });

describe("Homepage", () => {


        it('Should render correctly', function () {
            const component = renderer.create(<Homepage />);
            const tree = component.toJSON();
            expect(tree).toMatchSnapshot();
        });
    



})