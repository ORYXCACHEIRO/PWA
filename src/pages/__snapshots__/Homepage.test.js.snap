// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage Should render correctly 1`] = `
Array [
  <div
    className="ml-4 mr-4  relative aspect-[10/5] md:aspect-[10/6] sm:aspect-[9/13] flex items-start flex-col justify-center "
  >
    <div
      className="  relative ml-10 lg:ml-0"
    >
      <h1
        className=" font-bold text-gray-900  text-9xl  lg:text-8xl"
      >
        Looking for a place?
      </h1>
      <h1
        className="font-bold text-gray-900  text-9xl  lg:text-8xl"
      >
        Search 
        <span
          className="text-purple-500"
        >
          now!
        </span>
      </h1>
    </div>
    <div
      className="  h-max w-2/4 sm:w-full xl:w-3/4 mt-10 "
    >
      <form
        action="/search/:query"
        className="w-full flex"
        method="get"
      >
        <div
          className="w-full flex items-center justify-center"
        >
          <input
            className="border-3 border-purple-400 rounded-3xl h-14 w-4/5 ml-4 pl-3 font-bold lg:ml-16 md:ml-0 focus:outline-none focus:ring-3 focus:ring-purple-700 focus:border-transparent"
            name="s"
            placeholder="hotel..."
            type="text"
          />
          <button
            className="shadow-lg transition duration-300 ease-in-out bg-purple-900 font-bold text-lg p-3 pl-4 pr-4 rounded-3xl text-white m-5 hover:bg-gray-700 lg:mb-6"
            type="submit"
          >
            <svg
              className="h-6 w-6"
              fill="none"
              stroke="currentColor"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z"
                strokeLinecap="round"
                strokeLinejoin="round"
                strokeWidth="2"
              />
            </svg>
          </button>
        </div>
      </form>
    </div>
  </div>,
  <div
    className="ml-4 mr-4"
  >
    <div>
      <h1
        className="text-gray-900 m-5 ml-8 font-extrabold text-5xl text-center md:text-4xl"
      >
        Recomendations.
      </h1>
    </div>
    <div
      className="inline-flex pt-12 mr-auto ml-auto flex-wrap w-full justify-center gap-16 pb-10 lg:gap-12"
    />
  </div>,
]
`;
